### ISSUE TITLE
aws-runas should attempt to update or ignore environment variables

### ISSUE THREAD

**Issue by TopherGopher** [mmmorris1975/aws-runas#22]
With the 12h duration token, it works really nicely to run `` `aws-runas -d 12h qa` `` and have a shell with active credentials. This (as you know) uses my `~/.aws/config` file and generates export functions for the relevant AWS environment variables, eg
```
export AWS_REGION=''us-east-1''
export AWS_DEFAULT_REGION=''us-east-1''
export AWS_ACCESS_KEY_ID='<REDACTED>'
export AWS_SECRET_ACCESS_KEY='<REDACTED>'
export AWS_SESSION_TOKEN='<REDACTED>'
export AWS_SECURITY_TOKEN='<REDACTED>'
```
By injecting these in my shell, I can run tests and programs without having to worry about wrapping them with runas. Now here's where I start to run into problems.

#### What Happened
If I inject these into my shell and the next day I come back to the same shell and attempt to re-run the `` `aws-runas -d 12h qa` `` command, and I get an error:
```
$ aws-runas -d 12h qa
Enter MFA Code: 323851
aws-runas.main 2018/08/29 08:34:04 [CRIT] ▶ Error getting credentials: InvalidClientTokenId: The security token included in the request is invalid.
    status code: 403, request id: 93f9a497-ab98-11e8-a5d9-578cab764d57
```

It's attempting to use the security token that I cached in my environment yesterday, which fails for obvious reasons.

#### Expected Bahavior
If the environment variables are invalid, then runas should attempt to re-apply for them

#### Requested Fix/Feature
I would like to request that if the environment variables lead to a bad status code, then fall back to the file config, ignoring the environment and retry the request.

**Comment by mmmorris1975** [comment]
Hey Topher!  I have a possible fix for this in 1.1.2-beta2 if you wanted to try it out to see if it makes this work better

**Comment by mmmorris1975** [comment]
Fix released in 1.1.2
